ivy_tests/test_ivy/test_frontends/config/jax.py

```diff
@@ -71,3 +74,34 @@
     "complex128",
 ]
 invalid_complex_dtypes = []
+
+
+# Helpers for function testing
+
+
+Dtype = jax.dtypes.DType
+Device = jax.devices.Device
+
+
+def native_array(x):
+    return jnp.array(x)
+
+
+def is_native_array(x):
+    return isinstance(x, (jnp.ndarray, jnp.DeviceArray))
+
+
+def to_numpy(x):
+    return x
```

Comment:
We should return `np` arrays, this could be `jnp.ndarray` or `jnp.DeviceArray` which isn't correct.